omod/src/main/java/org/openmrs/module/shr/rest/web/controller/EncounterController.java

```diff
@@ -111,9 +107,14 @@ public Object postDocument(
 			if (log.isDebugEnabled()) {
 				log.debug("Request Response - " + error);
 			}
-			
 			response.setStatus(error.responseCode);
 			return error.response;
+		} catch (ContentHandlerException error){
```

Comment:
I mean why not allow it go higher up the chain and let Tomcat handle it, don't think we need this handling code. Moreover bad request suggest that this was an error on the part of the client - was it?